polling_stations/apps/data_collection/urls.py

```diff
@@ -1,10 +1,12 @@
 from django.conf.urls import url
-from django.urls import reverse_lazy
 from django.views.generic import RedirectView
 
-from .views import data_quality
 
 urlpatterns = [
-    url(r"^$", RedirectView.as_view(url=reverse_lazy("home")), name="league_table"),
-    url(r"data_quality/(?P<council_id>.+)/$", data_quality, name="data_quality"),
+    url(r"^$", RedirectView.as_view(pattern_name="home"), name="league_table"),
+    url(
+        r"data_quality/(?P<pk>.+)/$",
+        RedirectView.as_view(pattern_name="dashboard:council_detail"),
+        name="data_quality",
+    ),
```

Comment:
I don't really care if you want to bin this. If we ever fix/re-launch it, it needs re-thinking, but can you redirect the old route to the home page instead something that won't exist in production. Vaguely related discussion: https://github.com/DemocracyClub/UK-Polling-Stations/pull/1209

Reply:
I've now removed this `urls` module completely

Reply:
That just makes them both 404s though :(